cnames_active.js

```diff
@@ -2747,6 +2747,7 @@ var cnames_active = {
   "skratch": "websyncs.github.io/skratch",
   "sksamim": "esksamim.github.io/sksamim",
   "skyle": "skylejs.github.io/skyle",
+  "skyscript:" "cname.vercel-dns.com", //noCF
```

Comment:
```suggestion
  "skyscript": "cname.vercel-dns.com", // noCF
```
Your entry is currently invalid, malformed JS